templates/awslogs.conf.j2

```diff
@@ -63,5 +63,30 @@ file = /usr/share/tomcat/logs/manager.*.log
 log_group_name = {{ cldwatch_log_group }}
 log_stream_name = {{ hostname }}/manager_log
 datetime_format = %b %d %H:%M:%S
+
+{%   endif -%}
+{%   if ansible_env.HMPPS_ROLE == 'ndelius' or ansible_env.HMPPS_ROLE == 'spg' or ansible_env.HMPPS_ROLE == 'interfaces' -%}
```

Comment:
Can we look at using the in operator here to make it more readable

http://jinja.pocoo.org/docs/2.10/templates/#other-operators